Vagrantfile

```diff
@@ -19,6 +19,7 @@ echo 'cd ~/go/src/github.com/cilium/cilium' >> /home/vagrant/.bashrc
 SCRIPT
 
 $build = <<SCRIPT
+go get -u github.com/jteeuwen/go-bindata/...
```

Comment:
use `sudo -E env PATH="${PATH}" go get..` or use the `/usr/local/go/bin/go get...`